packages/airnode-examples/integrations/weather-multi-value/README.md

```diff
@@ -0,0 +1,27 @@
+# Authenticated, Multiple Encoded Value OpenWeather Example Integration
+
+This authenticated OpenWeather historical weather data example builds upon the foundations of the authenticated
+CoinMarketCap example by incorporating the encoding of multiple reserved parameters, which Airnode extracts and converts
+separately.
+
+As this is an authenticated example, you will need to sign up with OpenWeather in order to get a free API key. Like the
```

Comment:
1. I'd link `OpenWeather` to their homepage
2. I'd remove `Like the CoinMarketCap example` - I don't think this is necessary information for the reader

Reply:
Also, getting API key seems to be a bit difficult - at least I didn't mange to get a valid one...

When I try:
```
curl http://api.openweathermap.org/data/2.5/onecall/timemachine\&appid\=01b02c3e...
```

I get `{"cod":401, "message": "Invalid API key. Please see http://openweathermap.org/faq#error401 for more info."}`

When I try:

```
curl http://api.openweathermap.org/data/2.5/forecast\?id\=524901\&appid\=01b02c3...
```

it works.

Can you elaborate a bit more how to retrieve the API key for this particular API call? (Or it's just some stupid mistake on my side?)

Reply:
The following command works. Perhaps there was a delay in the API key being activated or something slightly off with the curl command itself (note: replace `MYKEY` with the API key and the quotes are necessary):
```sh
curl "https://api.openweathermap.org/data/2.5/onecall/timemachine?lat=51.507222&lon=-0.1275&dt=1637545002&appid=MYKEY"

Reply:
I addressed suggestions 1 and 2 in the latest commit. I also added the above curl command to the README so that users can ensure their API key is active (works on Windows cmd and Linux)